framework/wfa/ucc-dummy.py

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env python3
+
+import socket
+import argparse
+import sys
+import threading
+import binascii
+
+def handle(host, port, respDict):
+    id = threading.get_ident()
+    print("{}: Listening on {}:{}".format(id, host, port))
+    with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as s:
+        s.bind((host, port))
+        s.listen()
+        conn, addr = s.accept()
+        with conn:
+            print("{}: {}@{} Connected by {}".format(id, host, port, addr))
+            while True:
+                try:
+                    data = conn.recv(1024)
+                except Exception as e:
+                    print("Exception {}".format(e))
+                    break
+                print("{}@{} Received: {}".format(host, port, data))
+                resp = [b'Status,RUNNING', b'Status,COMPLETE'] # default response
+                for key in respDict:
+                    if key in str(data):

```

Comment:
That is a very roundabout way of working... I would do something more proper, like:
```
command_argv = str(data).strip().split(',')
command = command_argv[0].lower()
resp = respDict.get(command, resp)
```

Reply:
Not sure I understand to which lines you refer - can you add this with "insert suggestion"?

Reply:
Feel free to change that at any time then